Json-Rpc/Config.cs

```diff
@@ -29,6 +39,15 @@ public static void SetPreProcessHandler(PreProcessHandler handler)
         }
 
         /// <summary>
+        /// Sets the the PostProcessing Handler on the default session.
+        /// </summary>
+        /// <param name="handler"></param>
+        public static void SetPostProcessHandler(PostProcessHandler handler)
+        {
+            Handler.DefaultHandler.SetPostProcessHandler(handler);
+        }
```

Comment:
I've added support for this by making the method Set{Pre/Post}ProcessHandler public instead of internal, i hope that's ok?